### ISSUE TITLE
Question: How binance websocket on account channel is keep_alive?

### ISSUE THREAD

**Issue by xiandong79** [nardew/cryptoxlib-aio#63]
https://binance-docs.github.io/apidocs/spot/cn/#listen-key

```
Ping/Keep-alive a ListenKey (USER_STREAM)

    Response:

{}

PUT /api/v3/userDataStream

Keepalive a user data stream to prevent a time out. User data streams will close after 60 minutes. It's recommended to send a ping about every 30 minutes.
```

I see a function named "keep_alive_spot_listen_key()", however, after a global search, I did not find where it is used to keep account channel alive.

we can add a forever-loop which send a RESTful request every 30 min to keep account channel alive.

**Comment by nardew** [comment]
Currently the framework does not take care of refreshing the listenKey. As mentioned in #65, it makes sense to implement it.

**Comment by nardew** [comment]
Closed as the issue will be resolved together with issue #65.
